Dockerfile

```diff
@@ -0,0 +1,20 @@
+#
+# ci-fli-runner
+#
+
+# Apline 3.1 image.
+# https://hub.docker.com/_/alpine
+FROM alpine:3.10
+
+# Update package list.
+RUN apk update
+
+# Add bash package and dependencies.
+RUN apk add bash=5.0.0-r0
+
+# Add curl package and dependencies.
+RUN apk add curl=7.65.1-r0
```

Comment:
You may want to consider adding an additional step here to cleanup the package cache and reduce the resulting size of the image by performing `apk cache clean`.

Reply:
Hi Marc. Yes, I had originally tried this but got a "package cache not enabled" message, so removed it. I did look at manually clearing the cache (/var/cache/apk ?) but when I removed these files, it didn't reduce the image size at all. So I decided to leave them (not sure of the consequence of manually deleting).